### DISCUSSION TITLE
Não estou conseguindo instalar o Dapper

### DISCUSSION THREAD

**Discussion by rennanareas** [balta-io/forum#273] [Tira Dúvidas] [answered]
Estou tentando instalar o Dapper e está aparecendo esse erro e não estou sabendo como resolver. Alguém poderia me ajudar?

![erro](https://user-images.githubusercontent.com/55422641/220237997-78fe185a-5dd7-41dd-85b2-d11f814e8415.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @devrnx, é por que seu projeto também chama **Dapper**. Sugiro usar algum outro nome.
